Net Runner/ModelOutput/ImageNetClassificationModelOutput.mm

```diff
@@ -95,25 +95,27 @@ - (BOOL)isEqual:(id)anObject {
     
     NSAssert([previousOutput isKindOfClass:self.class], @"previousOutput is not same class as self: %@, %@", previousOutput.class, self.class);
     
-    const float decayValue = 0.75f;
-    const float updateValue = 0.25f;
+    const float decayValue = 0.70f;
+    const float updateValue = 0.30f;
     const float thresholdValue = 0.01f;
     
     NSMutableDictionary<NSString*,NSNumber*> *decayedInference = [NSMutableDictionary dictionary];
     NSDictionary<NSString*,NSNumber*> *previousInference = previousOutput.value[kClassificationOutputKey];
     NSDictionary<NSString*,NSNumber*> *newInference = self.value[kClassificationOutputKey];
     
     for ( NSString *key in previousInference ) {
-        decayedInference[key] = previousInference[key];
+        decayedInference[key] = @(previousInference[key].floatValue * decayValue);
```

Comment:
always apply decay. only applying decay to previous classifications that also appear in the latest classification was causing the stuck values